### PR TITLE
add archivista 0.7.0

### PR DESCRIPTION
**Pull request by wcrum**
Add Archivista.

### CODE REVIEW

File: packs/archivista-0.7.0/README.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+| service.type                               | string | `"ClusterIP"` |
+| tolerations                                | list   | `[]`          |
+
+
+## Reference
+
+* Helm chart: <https://github.com/in-toto/archivista/chart>
+* Archivista: <https://github.com/in-toto/archivista>
+
+### Maintainers
```
Comment (karl-cardenas-coding) [inline packs/archivista-0.7.0/README.md:52]: @wcrum can we move this into reference, as a link to the repo?

File: packs/archivista-0.7.0/README.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+| serviceAccount.name                        | string | `""`          |
+| service.port                               | int    | `8082`        |
+| service.type                               | string | `"ClusterIP"` |
+| tolerations                                | list   | `[]`          |
+
+
+## Reference
+
+* Helm chart: <https://github.com/in-toto/archivista/chart>
+* Archivista: <https://github.com/in-toto/archivista>
```
Comment (karl-cardenas-coding) [inline packs/archivista-0.7.0/README.md:50]: Let's make these bullets and use Markdown uRL

File: packs/archivista-0.7.0/README.md (hunk 1 above)
Comment (wcrum) [reply packs/archivista-0.7.0/README.md:52]: Maintainers is h3 while refernces is h2? Id say its under references, but where do you want it?

Comment (karl-cardenas-coding) [reply packs/archivista-0.7.0/README.md:52]: @wcrum Maintainers is not part of the template.

**Review by vishwanaths — APPROVED** [review]

**Review by karl-cardenas-coding — APPROVED** [review]
Approved from a docs perspective

**Pull request merged**